daemons/fenced/pacemaker-fenced.c

```diff
@@ -1546,12 +1556,30 @@ main(int argc, char **argv)
 
     pcmk__serve_fenced_ipc(&ipcs, &ipc_callbacks);
 
+    pcmk__register_formats(NULL, formats);
+    rc = pcmk__output_new(&out, "log", NULL, argv);
```

Comment:
Is the intent to use this in log_resource_details() instead of automatically creating one? Otherwise I don't see a point. Though looking at later commits, I wonder if you meant to drop this one

Reply:
The idea was that all the output messages in the scheduler would log instead of print to the screen when run via the daemons.

Reply:
That's a decent approach, and it means you don't need to create a separate logger in log_resource_details(), but it also means you can't use the existence of ->priv to distinguish crm_simulate from the daemons, and it means you need some sort of log level mechanism.

Reply:
Killing the separate logger shows new stuff getting added to the .scores files, so yeah definitely the right thing to do and also I clearly have some more work to do here.